samples/bluetooth/peripheral_fast_pair/src/main.c

```diff
@@ -17,28 +18,36 @@ LOG_MODULE_REGISTER(fp_sample, LOG_LEVEL_INF);
 #include "bt_adv_helper.h"
 #include "hids_helper.h"
 
-#define RUN_STATUS_LED			DK_LED1
-#define CON_STATUS_LED			DK_LED2
-#define FP_DISCOVERABLE_STATUS_LED	DK_LED3
+#define RUN_STATUS_LED				DK_LED1
+#define CON_STATUS_LED				DK_LED2
+#define FP_ADV_MODE_STATUS_LED			DK_LED3
 
-#define FP_DISCOVERABLE_BUTTON_MASK	DK_BTN1_MSK
-#define VOLUME_UP_BUTTON_MASK		DK_BTN2_MSK
-#define VOLUME_DOWN_BUTTON_MASK		DK_BTN4_MSK
+#define FP_ADV_MODE_BUTTON_MASK			DK_BTN1_MSK
+#define VOLUME_UP_BUTTON_MASK			DK_BTN2_MSK
+#define VOLUME_DOWN_BUTTON_MASK			DK_BTN4_MSK
 
-#define RUN_LED_BLINK_INTERVAL_MS	1000
+#define RUN_LED_BLINK_INTERVAL_MS		1000
+#define FP_ADV_MODE_LED_BLINK_INTERVAL_FAST_MS	500
+#define FP_ADV_MODE_LED_BLINK_INTERVAL_SLOW_MS	1500
```

Comment:
Maybe `FP_ADV_MODE_SHOW_UI_INDICATION_LED_BLINK_INTERVAL`? To make it more straightforward